app/assets/javascripts/discourse/app/models/topic-list.js

```diff
@@ -210,15 +212,19 @@ export default class TopicList extends RestModel {
   }
 
   // loads topics with these ids "before" the current topics
-  loadBefore(topic_ids, storeInSession) {
-    // refresh dupes
-    this.topics.removeObjects(
-      this.topics.filter((topic) => topic_ids.includes(topic.id))
-    );
+  async loadBefore(topic_ids, storeInSession) {
```

Comment:
I believe this function now needs to check if it's already loading data? (to prevent spamming clicks/triggering too many network requests)

Reply:
:+1: updated